src/lib/containers/EntityIdList.tsx

```diff
@@ -16,7 +16,7 @@ const EntityIdList: React.FC<EntityIdListProps> = props => {
   let mounted: boolean = true
 
   useEffect(() => {
-    if (mounted && inView) {
+    if (inView) {
```

Comment:
mounted will never be false because the effect won't be invoked after mounted is set to false in the cleanup.